<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lists</class>
 <widget class="QMainWindow" name="Lists">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lists</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/view-page-one-24.png</normaloff>:/images/view-page-one-24.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="departmentsToolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDepartments"/>
   <addaction name="actionJob_Titles"/>
   <addaction name="actionShifts"/>
   <addaction name="actionSaccos"/>
   <addaction name="actionHolidays"/>
   <addaction name="actionPayrollMonths"/>
  </widget>
  <action name="actionDepartments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1   Departments</string>
   </property>
   <property name="toolTip">
    <string>Departments (Alt+1)

Use this to set up the departments in the system.</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionJob_Titles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2   Job Titles</string>
   </property>
   <property name="toolTip">
    <string>Job Titles (Alt+2)

Use this to set up the job titles  in the system.</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionShifts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3   Shifts</string>
   </property>
   <property name="toolTip">
    <string>Shifts  (Alt+3)

Use this to set up the shifts in the system.</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionSaccos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4   SACCOs</string>
   </property>
   <property name="toolTip">
    <string>SACCOs (Alt+4)

Use this to set up the SACCOs in the system.</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionHolidays">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5   Holidays</string>
   </property>
   <property name="toolTip">
    <string>Holidays (Alt+5)

Use this to set up the holidays in the system.</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionPayrollMonths">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6   PayrollMonths</string>
   </property>
   <property name="toolTip">
    <string>Payroll Months</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
